libs/usb-drive/jest.config.js

```diff
@@ -5,5 +5,5 @@ const shared = require('../../jest.config.shared');
  */
 module.exports = {
   ...shared,
-  coveragePathIgnorePatterns: ['src/cli.ts'],
+  coveragePathIgnorePatterns: ['src/cli.ts', 'src/mocks'],
```

Comment:
The `memory_usb_drive` file was already ignoring coverage, but I'd rather have it here than in a file comment.